sonaranalyzer-dotnet/rspec/cs/S3011_c#.json

```diff
@@ -9,7 +9,7 @@
   "tags": [
     "owasp-a3"
   ],
-  "defaultSeverity": "Critical",
+  "defaultSeverity": "Minor",
```

Comment:
Indeed RSPEC-3011 is not related to security and it's a code smell but do we really want to make it minor?

Reply:
It has an impact on both code quality and performance as reflection is usually much slower.

Reply:
indeed, I will revert this as we won't change it now

but [using reflection](https://jira.sonarsource.com/browse/RSPEC-3011) should not be minor